patch/0018-i2c-mv64xxx-kdump-sonic.patch

```diff
@@ -0,0 +1,24 @@
+--- a/drivers/i2c/busses/i2c-mv64xxx.c	2025-05-14 17:43:42.421398200 +0300
```

Comment:
Please add the files created by `git format-patch`, that you also sent for review/upstreaming to the list.